pipeline.py

```diff
@@ -8,14 +8,22 @@
 from classifier import create_classifier
 import numpy as np
 import scipy.io
+import pickle
 
 
 def main():
-    raw, params = load_recordings("David2")
+    name = 'Haggai'
+    raw, params = load_recordings(name)
     raw = preprocess(raw)
     epochs, labels = get_epochs(raw, params["trial_duration"])
     features = get_features(epochs.get_data())
     clf, acc = create_classifier(features, labels)
+    classifier_num = 1
```

Comment:
Great job saving the classifier :) can we move this into a function `save_classifier` ?